sdk/template/azure-template/src/template_client.cpp

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
+// Just Triggering Testing Pipelines
```

Comment:
I'm surprised this can trigger a code review update - the code review json files *should* be idempotent and the only lines that would change (the #line directives) should be marked as "exclude from diff".